### ISSUE TITLE
Slå sammen Detektorer og Sensorer

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#576]
Hva tror du, @mittinatten ?

Jeg er fristet til å slå disse sammen:
- https://app.uio.no/ub/emnesok/realfagstermer/search?id=c007500
- https://app.uio.no/ub/emnesok/realfagstermer/search?id=c011761

men ser at jeg må omklasse litt fra `Detektorer` til `Partikkeldetektorer` isåfall. 

Noe sånt?

```
Sensorer
BF Detektorer
SO Partikkeldetektorer
```

```
Partikkeldetektorer
SO Sensorer
BF Detektorer for radioaktiv stråling
BF Detektorer for ioniserende stråling
```

**Comment by mittinatten** [comment]
Ja, det burde gå bra hvis en skiller ut partikkeldetektorer. Legge til `UT Scintillatorer` og `BF Geigerteller` på den siste.

**Comment by danmichaelo** [comment]
Som nevnt i #641 har vi også Kjemiske/optiske sensorer/detektorer.
